api/src/main/java/org/pentaho/platform/api/genericfile/model/IGenericFileContentWrapper.java

```diff
@@ -0,0 +1,13 @@
+package org.pentaho.platform.api.genericfile.model;
+
+import java.io.InputStream;
+
+public interface IGenericFileContentWrapper {
```

Comment:
I'm normally don't care about code names or method names, but adding "Wrapper" seem overkill.

It's fairly common that an inputstream will accept another input stream as an argument and by implied it's a wrapper.

https://docs.oracle.com/javase%2F8%2Fdocs%2Fapi%2F%2F/java/io/BufferedInputStream.html#BufferedInputStream-java.io.InputStream-

https://commons.apache.org/proper/commons-vfs/commons-vfs2/apidocs/org/apache/commons/vfs2/util/MonitorOutputStream.html#MonitorOutputStream-java.io.OutputStream-